app/src/main/java/com/amaze/filemanager/ui/activities/superclasses/PermissionsActivity.java

```diff
@@ -60,7 +60,7 @@
   public void onRequestPermissionsResult(
       int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
     super.onRequestPermissionsResult(requestCode, permissions, grantResults);
-    if (requestCode == STORAGE_PERMISSION) {
+    if (requestCode == STORAGE_PERMISSION || requestCode == ALL_FILES_PERMISSION) {
```

Comment:
This will never happen I think. The all files permission is started as an activity, it isn't a permission dialog, and it doesn't expect a permission result. 
That said, can you remove line no. 197, as we don't enable the rotation again in all files permission.